.gitignore

```diff
@@ -21,6 +21,7 @@ interactive_otp_main.json
 /otp-ui
 Dockerfile
 .java-version
+o_o_standalone_config_IncludeFileDirectiveTest_part.json
```

Comment:
Doesn't hurt but what is this?

Reply:
Not sure what it is, but a file with that name is created in the top directory every time you run maven test on Windows. Its not even a valid JSON file.